content/F2016/F16-Lunch-with-Microsoft.md

```diff
@@ -0,0 +1,18 @@
+title: Lunch With Microsoft
+Date: 2016-9-26
+Category: Events
+Tags: social, volunteer, social
+Slug: lunch-with-microsoft
+Author: Shivangi Patwardhan
+Summary: Come join Microsoft for lunch and meet current full-times!
+
+Come join Microsoft for lunch and meet current full-times! 
+Got class in-between? No worries, drop by anytime between 12:30-1:30pm to join us for Taco Farm.
+We will be joined by Antika Dey, Software Engineer in Product Ads; Kyle Olive, Program Manager on Windows Internet of Things; Sage Franch, Tech Evagelist and founder and TrendyTechie.ca.
+
```

Comment:
- software engineer, program manager, etc should be all lowercase
- this is a nitpicky thing but I'm not sure that values should be separated by semi-colons. "Ankita Dey (software engineer), Kyle Oline (program manager on Windows Internet of Things)...."